pkg/provider/kind/kind.go

```diff
@@ -209,3 +210,43 @@ func (c *KIND) GetDeploymentVars(parseContext *kingpin.ParseContext) error {
 	}
 	return nil
 }
+
+// Get Prombench services
+func (c *KIND) GetKINDService(*kingpin.ParseContext) error {
+
+	var (
+		InternalIP string
+		NodePort   int32
+	)
+
+	pods, err := c.k8sProvider.Clt.CoreV1().Pods("default").List(c.ctx, metav1.ListOptions{LabelSelector: "app=grafana"})
+	if k8sErrors.IsNotFound(err) || err != nil {
+		return err
+	}
+
+	for _, v := range pods.Items {
+		node, err := c.k8sProvider.Clt.CoreV1().Nodes().Get(c.ctx, v.Spec.NodeName, metav1.GetOptions{})
+		if k8sErrors.IsNotFound(err) || err != nil {
+			return err
+		}
+
+		for _, v := range node.Status.Addresses {
+			if v.Type == "InternalIP" {
+				InternalIP = v.Address
+			}
+		}
+	}
+
+	service, err := c.k8sProvider.Clt.CoreV1().Services("default").Get(c.ctx, "grafana", metav1.GetOptions{})
+	if k8sErrors.IsNotFound(err) || err != nil {
+		return err
+	}
+
+	NodePort = service.Spec.Ports[0].NodePort
+
+	fmt.Printf("Grafana: http://%v:%v/grafana\n", InternalIP, NodePort)
```

Comment:
Rather than printing the services I think we should just simply print the InternalIP and NodePort and one could use the README to see whatever service the want. :) We primarily want the InternaIP and NodePrort here so.